### PR TITLE
Update data science learning path and update new-contributors.md

### PR DESCRIPTION
**Pull request by KtrauM**

### CODE REVIEW

**Comment by dipakkr** [comment]
hi @KtrauM please update your PR. there is some conflict.

File: new-contributors.md [hunk 1]
```diff
@@ -41,6 +41,6 @@
 | 35. | Suyasha | New York City | | [GitHub](https://github.com/suyasha0)|
 | 36. | Thomas Smith | Ireland | Salesforce Developer | [GitHub](https://github.com/ThomasSmithIRE) - [LinkedIn](https://www.linkedin.com/in/engineertsmith/) |
 |37. | Ken Davis | - | Web Development | [GitHub](https://github.com/kdavis-ssi) - [LinkedIn](https://www.linkedin.com/in/ken-davis-9a6a8051/) -[Twitter](http://twitter.com/kdavis158)| 
-
```
Comment (dipakkr) [inline new-contributors.md]: update your PR. there are some conflict to due to new changes.

Comment (KtrauM) [reply new-contributors.md]: What kind of conflict ? Should I just add my personal information to new-contributers.md again or to CONTRIBUTORS.md ?

Comment (dipakkr) [reply new-contributors.md]: just add to CONTRIBUTORS.md. first update your master branch from remote.

**Pull request closed without merging**